sim/Makefile

```diff
@@ -34,6 +34,21 @@ base_dir := $(firesim_base_dir)
 chipyard_dir := $(abspath ..)/target-design/chipyard
 rocketchip_dir := $(chipyard_dir)/generators/rocket-chip
 
+else
+# Chipyard make variables
+base_dir := $(abspath ../../..)
+sim_dir := $(firesim_base_dir)
+chipyard_dir := $(base_dir)
+include $(base_dir)/variables.mk
+# target-agnostic.mk wants lower-case rocketchip_dir but CY defines it upper-case
+rocketchip_dir := $(ROCKETCHIP_DIR)
```

Comment:
This is needed so that I can remove hardcoded `rocketchip_dir` from `target-agnostic.mk` below.  It is overriden in my TARGET_PROJECT_MAKEFRAG to use my toplevel project's rocket-chip.

Reply:
I'm okay with changing it or leaving it like this.  I just want to be able to define it in `sim/Makefile` and get rid of the re-definition in `sim/target-agnostic.mk`.